test/integration2/schema/views.spec.js

```diff
@@ -122,6 +122,95 @@ describe('Views', () => {
             });
         });
 
+        it('create or replace view', async () => {
+          await knex.schema
```

Comment:
can you create view with same name before, test it exists, then make this test to really test that view is really replaced.